connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainer.kt

```diff
@@ -227,7 +227,10 @@ internal class StripeConnectWebViewContainerImpl<Listener, Props>(
     }
 
     internal fun setPropsFromXml(props: Props) {
-        this.propsJson = props.toJsonObject()
+        // Only set props if uninitialized.
+        if (controller == null) {
```

Comment:
Good catch! So basically we should only set props once, and if we've already set props (ex. from the programmatic case), we should not set it here (in the XML case)